src/herder/TransactionQueue.cpp

```diff
@@ -240,14 +244,14 @@ TransactionQueue::tryAdd(TransactionFrameBasePtr tx)
 
     if (oldTxIter != stateIter->second.mTransactions.end())
     {
-        releaseFeeMaybeEraseAccountState(*oldTxIter);
-        stateIter->second.mQueueSizeOps -= (*oldTxIter)->getNumOperations();
-        mQueueSizeOps -= (*oldTxIter)->getNumOperations();
-        *oldTxIter = tx;
+        releaseFeeMaybeEraseAccountState(oldTxIter->mTx);
+        stateIter->second.mQueueSizeOps -= oldTxIter->mTx->getNumOperations();
+        mQueueSizeOps -= oldTxIter->mTx->getNumOperations();
+        *oldTxIter = {tx, mApp.getClock().now()};
```

Comment:
I'm unsure about this. Is this conceptually what we are trying to measure here? Specifically, this allows the slightly unusual property that the timestamps for a single account might not be monotonically increasing. Do we want to measure "how long did it take for specific transactions to get included" or "how long did it take for specific combination of source account and sequence number to get included"?

Reply:
per the issue and metric definition, it's tracking how long a specific transaction has been sitting in the queue. The notion of "sequence number slot" is irrelevant per this definition.